requirements.txt

```diff
@@ -13,3 +13,4 @@ pdmongo
 lz4
 immutabledict
 packaging
+mongomock
```

Comment:
I dont think mongomock should be a requirement in the regular install, rather only for development.

Reply:
I have to add `mongomock.MongoClient` to `corrections.py` [here](https://github.com/ahiguera-mx/strax/blob/eef44093654055eb19970d98b0854f0f2667999e/strax/corrections.py#L52), if I remove mongomock, this would break, right?

Reply:
you can use mongomock.patch() in the test instead of having mongomock.MongoClient in the codebase itself.